src/pages/understand-stacks/stacking-using-CLI.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Stacking using the Stacks CLI
+description: Learn how use the Stacks CLI to participate in stacking
+experience: beginner
+duration: 10 minutes
+tags:
+  - guide
+images:
+  sm: /images/pages/stacking-rounded.svg
+---
+
+## Introduction
+
+!> The Stacking implementation is still in development and could change in the coming weeks
+
+In this guide, you'll learn how to use the Stacks CLI to participate in stacking. The CLI is a great way to quickly try out stacking on testnet. To integrate stacking into your application, see the [stacking integration guide](/stacks-blockchain/integrate-stacking).
```

Comment:
- "In this tutorial"
- I believe we adopted the convention to write "Stacking" to indicate it is a product
- "see the stacking integration tutorial"